src/librustc_typeck/diagnostics.rs

```diff
@@ -4826,6 +4826,10 @@ type, it's not allowed to override anything in those implementations, as it
 would be ambiguous which override should actually be used.
 "##,
 
+E0720: r##"
+A `union` can not have fields with destructors.
+"##,
+
```

Comment:
Do we have a rule behind how to assign error numbers? Yes, the text is a bit lacking here.

Reply:
We do not really have a system, no. You just pick one.